proto/booking/cancellation_response.proto

```diff
@@ -8,4 +8,5 @@ enum CancellationResponse {
     SUCCESSFUL = 0;
     MISMATCHED_ESTIMATE = 1;
     ALREADY_CANCELLED = 2;
+	INVALID_STATE = 3;
```

Comment:
Can you make this "INVALID_RESERVATION_STATE", then go ahead and merge it?

Good to see this getting added, but "INVALID_STATE" is just a wee too general, and non-descriptive, for my taste.